README.md

```diff
@@ -239,17 +240,17 @@ X-Cache: HIT from url-to-your-proxy
 
 Example with the Norwegian Stratum 1:
 ```
-curl --head http://bgo-no.stratum1.cvmfs.eessi-infra.org/cvmfs/pilot.eessi-hpc.org/.cvmfspublished
+curl --head http://bgo-no.stratum1.cvmfs.eessi-infra.org/cvmfs/software.eessi.io/.cvmfspublished
```

Comment:
This doesn't work for `software.eessi.io`. Just a configuration issue of the stratum 1?

Reply:
Well spotted! Fixed it by using a new Stratum 1.